cdk/bin/cdk.ts

```diff
@@ -1,7 +1,27 @@
 import "source-map-support/register";
 import { App } from "aws-cdk-lib";
+import type { SupportRemindersProps } from "../lib/support-reminders";
 import { SupportReminders } from "../lib/support-reminders";
 
 const app = new App();
-new SupportReminders(app, "SupportReminders-CODE", { stack: "support", stage: "CODE" });
-new SupportReminders(app, "SupportReminders-PROD", { stack: "support", stage: "PROD" });
+
+export const codeProps: SupportRemindersProps = {
+	stack: "support",
+	stage: "CODE",
+	certificateId: "b384a6a0-2f54-4874-b99b-96eeff96c009",
+	domainName: "reminders-code.support.guardianapis.com",
+	hostedZoneId: "Z3KO35ELNWZMSX",
+	datalakeBucket: "contributions-private",
+};
+
+export const prodProps: SupportRemindersProps = {
+	stack: "support",
+	stage: "PROD",
+	certificateId: "b384a6a0-2f54-4874-b99b-96eeff96c009",
```

Comment:
It looks like we are using the same value for `certificateId` in `CODE` and `PROD`. IIRC we typically use different certificates for each stage, so just wanted to check that this is expected/desirable in this case?

Reply:
Yeah, both `CODE` and `PROD` use the `support.guardianapis.com` domain, and so both use the same certificate. As far as I know this is how this repo has always been configured, for better or worse. As @akash1810 suggested, I'm going to change the certificate config by provisioning certificates as part of the CFN stack instead of referencing an existing certificate.